lib/tds.js

```diff
@@ -278,9 +297,9 @@ async function generateDNRRulesForTrackerEntry (
         if (surrogate) {
             resourceTypes = ['script']
             if (!supportedSurrogateScripts.has(surrogate)) {
-                // Block requests if an unsupported surrogate script rule
-                // matches.
-                ruleAction = 'block'
+                // we should ignore actions for missing surrogates, otherwise breakage can result?
```

Comment:
Yea, I did it that way initially, but then I found the expected behavior was to block. [See this reference test](https://github.com/duckduckgo/privacy-reference-tests/blob/main/tracker-radar-tests/TR-domain-matching/domain_matching_tests.json#L260).

Reply:
👍  fixed